amazon-sqs/5.11/modules/ROOT/pages/index.adoc

```diff
@@ -37,7 +37,21 @@ Before creating an app, you must:
 
 == Common Use Cases for the Connector
 
-Common use cases for Amazon SQS Connector include sending messages to a queue and retrieving messages from a queue. For an example of these use cases, see xref:amazon-sqs-connector-examples.adoc#install-connector[Amazon SQS Examples].
+Common use cases for Amazon SQS Connector include:
+
```

Comment:
Include a use case header and description for sending messages to a queue and retrieving messages from a queue as the first use case in the beginning
ex; 
Queue Messaging
Send messages to a queue and retrieve messages from a queue